cookbooks/simple_ice_box.prm

```diff
@@ -0,0 +1,174 @@
+# A simple setup for using the boundary heat flux in a ice viscoplatic model.
+
+
+set Dimension                              = 2
+set Use years in output instead of seconds = true
+set End time                               = 1e6
+set Output directory                       = output-ice-box_2d
+set Nonlinear solver scheme                = single Advection, iterated Stokes 
+set Nonlinear solver tolerance             = 1e-5
+set Max nonlinear iterations               = 10
```

Comment:
Typically we set this value to a much higher number (50 or 100). In general, we never want to reach this limit (e.g., the model should always converge). Often, the limit is only used to ensure the code is not stuck doing an endless number of non-linear iterations. 

For now, I suggest changing this value to 50 and see if that is a sufficient number of non-linear iterations to achieve convergence. If not, I think this would be a great application for the Newton solver, which is enabled with `set Nonlinear solver scheme   = iterated Advection, Newton Stokes`

Reply:
We can talk more about the different nonlinear solver schemes later on.